src/libMetrics/Metrics.h

```diff
@@ -22,13 +22,16 @@
 #include <list>
 #include <string>
 
+#ifndef HAVE_CPP_STDLIB
```

Comment:
Note for @Yaron:
This was required for the latest open telemetry, not having it caused some very nasty memory issues.

Reply:
OK then please do the official way after this is merged, life was very painful with this bug.